prow/plugins/golint/golint.go

```diff
@@ -232,6 +251,9 @@ func handle(minimumConfidence float64, ghc githubClient, gc *git.Client, log *lo
 
 	// Make the list of comments.
 	var comments []github.DraftReviewComment
+	if lintErrorComment != (github.DraftReviewComment{}) {
```

Comment:
nit: Please make `problemsInFiles` return a ~pointer to a comment struct~ slice of comments so that we can do a `nil` check here instead of comparing against an empty struct and also provide comments for all files that can't be linted instead of just one.